### ISSUE TITLE
[eslint-plugin-react-hooks] Compatibility with ESLint 6.0.0

### ISSUE THREAD

**Issue by tbroyer** [facebook/react#15971]
**Do you want to request a *feature* or report a *bug*?**
feature

**What is the current behavior?**
Using in my `package.json`:
```
    "eslint": "^6.0.0",
    "eslint-plugin-react-hooks": "^1.6.0",
```
Yarn emits:
```
warning " > eslint-plugin-react-hooks@1.6.0" has incorrect peer dependency "eslint@^3.0.0 || ^4.0.0 || ^5.0.0".
```

**What is the expected behavior?**
No warning.

AFAICT, there shouldn't be other change needed besides updating the peer dependency version in the `package.json`: https://eslint.org/docs/user-guide/migrating-to-6.0.0
so it looks like I could just ignore the warning, but I'd rather be sure the plugin is actually compatible with ESLint 6.0.0

**Which versions of React, and which browser / OS are affected by this issue? Did this work in previous versions of React?**
n/a

**Comment by gaearon** [comment]
Wanna send a PR?

**Comment by tbroyer** [comment]
Done #15974

**Comment by gaearon** [comment]
Released as 1.6.1. Thanks.
